projections/customer.go

```diff
@@ -0,0 +1,50 @@
+package projections
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/example/messages/events"
+)
+
+// CustomerProjectionHandler is a projection that builds a report of customers
+// acquired by the bank.
+type CustomerProjectionHandler struct {
+	dogma.NoTimeoutHintBehavior
+}
+
+// Configure configs the engine for this projection.
+func (h *CustomerProjectionHandler) Configure(c dogma.ProjectionConfigurer) {
+	c.Identity("customer-projection", "70b58269-931a-46d6-b745-286a670fb6f7")
```

Comment:
I know the existing example had this `-projection` suffix in its name, but I think we should try to think of a way to name these that *doesn't* require that suffix... is there a more specific name we can give it?

I won't hold up the PR on this, but if you think of anything LMK.

Reply:
It is supposed to be the projection for the select customer UI. Should any of that be used as part of the name?

Reply:
While that will be the only use-case for it in this repo, I think it's probably a mistake to present it that way as an example of how to use Dogma.

In a "real" system you would likely have some projections that are quite closely tied to the core domain, and others that really are UI-specific. I consider this to be the former.

Let's leave it as is for now; it's not an issue to change it in the future.